apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/Trace/TraceHeaderContext.cs

```diff
@@ -28,6 +28,8 @@ namespace Google.Cloud.Diagnostics.Common
     /// </remarks>
     internal sealed class TraceHeaderContext
     {
+        private static readonly TraceIdFactory _traceIdFactory = TraceIdFactory.Create();
```

Comment:
How much value is TraceIdFactory providing as a class? Would a single static method in this class do just as well? Just a thought.